website/src/components/BlogListItem.tsx

```diff
@@ -9,6 +9,7 @@ interface Blog {
     description: string
     type: string
     href: string
+    alt: string
     image?: string
```

Comment:
I'd suggest this:

```ts
interface Image {
  src: string
  alt: string
}

image?: Image
```

Reply:
Ah, yea `image` is optional, this makes more sense. Updated!